### ISSUE TITLE
Crash when close the chat or open closed chat

### ISSUE THREAD

**Issue by artemmihelson** [helpcrunch/helpcrunchsdk-ios#16]
Hi, I got crash on closed chat

```
2022-11-17 16:57:00.218145+0200 MADev[1002:6449987] *** Assertion failure in -[UICollectionView _dequeueReusableViewOfKind:withIdentifier:forIndexPath:viewCategory:], UICollectionView.m:8556
2022-11-17 16:57:00.293399+0200 MADev[1002:6449987] *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'could not dequeue a view of kind: UICollectionElementKindCell with identifier TechMessage - must register a nib or a class for the identifier or connect a prototype cell in a storyboard'
*** First throw call stack:
(
	0   CoreFoundation                      0x0000000117f228cb __exceptionPreprocess + 242
	1   libobjc.A.dylib                     0x00000001139cfba3 objc_exception_throw + 48
	2   Foundation                          0x00000001160aa37c _userInfoForFileAndLine + 0
	3   UIKitCore                           0x000000013289b73a -[UICollectionView _dequeueReusableViewOfKind:withIdentifier:forIndexPath:viewCategory:] + 2681
	4   UIKitCore                           0x000000013289b8bd -[UICollectionView dequeueReusableCellWithReuseIdentifier:forIndexPath:] + 88
	5   HelpCrunchSDK                       0x0000000113083a76 -[HCSChatDataSource collectionView:cellForItemAtIndexPath:] + 438
	6   UIKitCore                           0x0000000132880371 -[UICollectionView _createPreparedCellForItemAtIndexPath:withLayoutAttributes:applyAttributes:isFocused:notify:] + 1262
	7   UIKitCore                           0x000000013287fe7d -[UICollectionView _createPreparedCellForItemAtIndexPath:withLayoutAttributes:applyAttributes:] + 31
	8   UIKitCore                           0x000000013288a3d1 -[UICollectionView _createVisibleViewsForSingleCategoryAttributes:limitCreation:fadeForBoundsChange:] + 1731
	9   UIKitCore                           0x000000013288a8d5 -[UICollectionView _createVisibleViewsForAttributes:fadeForBoundsChange:notifyLayoutForVisibleCellsPass:] + 376
	10  UIKitCore                           0x0000000132888637 -[UICollectionView _updateVisibleCellsNow:] + 2331
	11  UIKitCore                           0x000000013288f24e -[UICollectionView layoutSubviews] + 375
	12  UIKitCore                           0x00000001339ad433 -[UIView(CALayerDelegate) layoutSublayersOfLayer:] + 2305
	13  QuartzCore                          0x00000001153f10fc _ZN2CA5Layer16layout_if_neededEPNS_11TransactionE + 526
	14  QuartzCore                          0x00000001153fc5d5 _ZN2CA5Layer28layout_and_display_if_neededEPNS_11TransactionE + 65
	15  QuartzCore                          0x000000011530e431 _ZN2CA7Context18commit_transactionEPNS_11TransactionEdPd + 623
	16  QuartzCore                          0x0000000115345d56 _ZN2CA11Transaction6commitEv + 714
	17  QuartzCore                          0x00000001153471e4 _ZN2CA11Transaction25flush_as_runloop_observerEb + 60
	18  CoreFoundation                      0x0000000117e810a5 __CFRUNLOOP_IS_CALLING_OUT_TO_AN_OBSERVER_CALLBACK_FUNCTION__ + 23
	19  CoreFoundation                      0x0000000117e7ba12 __CFRunLoopDoObservers + 515
	20  CoreFoundation                      0x0000000117e7bf5d __CFRunLoopRun + 1161
	21  CoreFoundation                      0x0000000117e7b6f7 CFRunLoopRunSpecific + 560
	22  GraphicsServices                    0x000000011b5ec28a GSEventRunModal + 139
	23  UIKitCore                           0x000000013337f62b -[UIApplication _run] + 994
	24  UIKitCore                           0x0000000133384547 UIApplicationMain + 123
	25  MADev                               0x0000000104c935ff main + 63
	26  dyld                                0x000000010ee422bf start_sim + 10
	27  ???                                 0x0000000206ad3310 0x0 + 8701948688
)
libc++abi: terminating with uncaught exception of type NSException
Message from debugger: Terminated due to signal 6
```

**Comment by Fishkiller** [comment]
@artemmihelson thanks for the pointing out! It is fixed, please check latest 4.2.3 version

**Comment by artemmihelson** [comment]
@Fishkiller yep, confirm it is fixed. Closing this issue.
